scripts/test_updates_pg12.sh

```diff
@@ -20,3 +20,5 @@ run_tests "$@" -v7 \
           2.0.0-rc2-pg12 2.0.0-rc3-pg12 2.0.0-rc4-pg12 2.0.0-pg12 2.0.1-pg12 2.0.2-pg12 2.1.0-pg12 \
           2.1.1-pg12 2.2.0-pg12 2.2.1-pg12 2.3.0-pg12 2.3.1-pg12 2.4.0-pg12 2.4.1-pg12 2.4.2-pg12 \
           2.5.0-pg12 2.5.1-pg12
+run_tests "$@" -v8 \
+          2.5.0-pg12 2.5.1-pg12
```

Comment:
Since you are adding a test for enabling compression on cagg which is only possible after this PR is merged the earliest v8 can run is 2.6.0

Reply:
The test is in post.continuous_aggs - checks if we can enable compression on an existing  cagg , after upgrading to the latest version.Removed -v7 test from 2.5.0 and 2.5.1 since v8 is a superset of v7.